src/index.html

```diff
@@ -9,10 +9,59 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      type="text/css"
+      href="style.css"
```

Comment:
The HTML file links to a single `style.css` file. According to the task requirements, each BEM block should have its own separate CSS file. Please organize the styles into separate files for each BEM block and update the HTML to link to these files.